exampleCourse/questions/demo/annotated/engLogic/question.html

```diff
@@ -9,28 +9,34 @@
 <pl-hide-in-panel answer="true">
 <pl-overlay width="500" clip="false">
     <pl-background>
-      <pl-figure file-name="logic-diagram.png" directory="clientFilesQuestion"></pl-figure>
+      <div id="logic-diagram">
+        <pl-figure
+        file-name="logic-diagram.png"
+        alt="A three-input logic diagram. The top line, x, feeds through an arrowhead and into the output F. The middle line, y, feeds through a triangle with a circle on the right, then a semicircle. The bottom line, z, feeds into the semicircle. The semicircle feeds through an arrowhead into the output F."
+        directory="clientFilesQuestion"
```

Comment:
Nit (keeping information about the file's location adjacent to each other):

```suggestion
        directory="clientFilesQuestion"
        alt="A three-input logic diagram. The top line, x, feeds through an arrowhead and into the output F. The middle line, y, feeds through a triangle with a circle on the right, then a semicircle. The bottom line, z, feeds into the semicircle. The semicircle feeds through an arrowhead into the output F."
```

Reply:
[3f527d0](https://github.com/PrairieLearn/PrairieLearn/pull/11844/commits/3f527d0bd67b8c09d9e8416a41c6855d08e6f262)